flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/AbstractJsonDeserializationSchema.java

```diff
@@ -155,4 +161,14 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(failOnMissingField, ignoreParseErrors, resultTypeInfo, timestampFormat);
     }
+
+    /**
+     * Logs a debug message when a JSON parse error is ignored.
+     *
+     * @param message the original JSON message that failed to parse
+     * @param t the throwable that was caught
+     */
+    protected void logParseError(byte[] message, Throwable t) {
+        LOG.debug("Failed to deserialize JSON '{}'.", new String(message), t);
```

Comment:
```suggestion
        if (LOG.isDebugEnabled()) {
            LOG.debug("Failed to deserialize JSON '{}'.", new String(message), t);
        }
```

What about using pre-judge for logger level ?
Because we could reduce one redundant bytes array allocation if the logger level is not enabled debug or lower levels